src/validator/SwaggerValidator.ts

```diff
@@ -2,5 +2,6 @@ import {OpenAPIV2} from 'openapi-types';
 import SwaggerParser from 'swagger-parser';
 
 export async function validateConfig(swaggerJson: OpenAPIV2.Document): Promise<void> {
-  await SwaggerParser.validate(swaggerJson);
+  const swaggerJsonCopy = JSON.parse(JSON.stringify(swaggerJson));
```

Comment:
How about referencing [the issue](https://github.com/APIDevTools/swagger-parser/issues/77) here so that we remember why we create a copy?